timesheet_helpdesk_ticket/views/helpdesk_team.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<data>
+    <record id="timesheet_helpdesk_team_view_form" model="ir.ui.view">
+        <field name="name">timesheet.helpdesk.team.form.view</field>
+        <field name="model">helpdesk.ticket.team</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.view_helpdesk_team_form" />
+        <field name="priority" eval="20" />
+        <field name="arch" type="xml">
+            <!-- Only For Fix Smart Button-->
+            <xpath expr="//div[@name='button_box']" position="replace">
```

Comment:
Off topic.
This has to be a PR on heldesk_mgmt

Reply:
Fixed it.